### PR TITLE
[CP-1624] Adding numberID as a field to contact and message response

### PR DESCRIPTION
**Pull request by lukasmas**
Changing the way Center is recognizing the threads in the message window by introducing the numberID.

**Your checklist for this pull request**

Make sure that this PR:
- [x] Complies with our guidelines for contributions
- [ ] Has unit tests if possible
- [x] Has documentation updated
- [x] Has changelog entry added

### CODE REVIEW

File: doc/os_api/endpoints/pure/contacts_endpoint.md [hunk 1]
```diff
@@ -1,103 +1,117 @@
...
+        "address": "Czeczota 6, 02-600 Warsaw, Poland",
+        "altName": "Abraham",
+        "email": "Abraham@example.com",
+        "blocked": false,
+        "favourite": true,
+        "ice": true,
+        "id": 1,
+        "numbers": [
+          "123456789"
+        ],
```
Comment (dkarski) [inline doc/os_api/endpoints/pure/contacts_endpoint.md:86]: `numbersIDs` is missing here 😉

File: doc/os_api/endpoints/pure/messages_endpoint.md [hunk 2]
```diff
@@ -0,0 +1,773 @@
...
+
+```json
+{
+  "body": {
+    "isUnread": false,
+    "lastUpdatedAt": 1650656989,
+    "messageCount": 1,
+    "messageSnippet": "Hello",
+    "messageType": 1,
+    "threadID": 8
```
Comment (dkarski) [inline doc/os_api/endpoints/pure/messages_endpoint.md:157]: `numberID` is missing here 😉

File: doc/os_api/endpoints/pure/messages_endpoint.md [hunk 3]
```diff
@@ -0,0 +1,773 @@
...
+{
+  "body": {
+    "entries": [
+      {
+        "messageBody": "nieudane wysy<0xc5><0x82>anie :(",
+        "messageID": 6,
+        "messageType": 2,
+        "createdAt": 1547492320,
+        "threadID": 1,
+        "number": "345678912"
```
Comment (dkarski) [inline doc/os_api/endpoints/pure/messages_endpoint.md:284]: `numberID` is missing here

File: doc/os_api/endpoints/pure/messages_endpoint.md [hunk 4]
```diff
@@ -0,0 +1,773 @@
...
+
+```json
+{
+  "body": {
+    "messageBody": "heh?",
+    "messageID": 8,
+    "messageType": 4,
+    "createdAt": 1547492320,
+    "threadID": 2,
+    "number": "123456789"
```
Comment (dkarski) [inline doc/os_api/endpoints/pure/messages_endpoint.md:363]: `numberID` is missing here

File: doc/os_api/endpoints/pure/messages_endpoint.md [hunk 5]
```diff
@@ -0,0 +1,773 @@
...
+{
+  "body": {
+    "entries": [
+      {
+        "messageBody": "najstarsze odb fsjdklafjskldjf",
+        "messageID": 1,
+        "messageType": 4,
+        "createdAt": 1547465101,
+        "threadID": 1,
+        "number": "123456789"
```
Comment (dkarski) [inline doc/os_api/endpoints/pure/messages_endpoint.md:401]: `numberID` is missing here

File: doc/os_api/endpoints/pure/contacts_endpoint.md (hunk 1 above)
Comment (lukasmas) [reply doc/os_api/endpoints/pure/contacts_endpoint.md:86]: added

File: doc/os_api/endpoints/pure/messages_endpoint.md (hunk 2 above)
Comment (lukasmas) [reply doc/os_api/endpoints/pure/messages_endpoint.md:157]: number ID is not needed here

**Review by dkarski — APPROVED** [review]
LGTM ✅ _(from the documentation perspective)_

File: module-db/Interface/ContactRecord.cpp [hunk 6]
```diff
@@ -1295,28 +1295,31 @@ auto ContactRecordInterface::getNumbers(const std::string &numbers_id) -> std::v
         try {
             auto &&number = nr.numbere164.empty() ? utils::PhoneNumber(nr.numberUser, utils::country::Id::UNKNOWN)
                                                   : utils::PhoneNumber(nr.numberUser, nr.numbere164);
-            nrs.emplace_back(number.getView(), nr.type);
+            nrs.emplace_back(number.getView(), nr.type, nr_val);
```
Comment (SP2FET) [inline module-db/Interface/ContactRecord.cpp:1298]: please stay with camelCase :P

**Review by SP2FET — APPROVED** [review]

File: module-db/Interface/ContactRecord.cpp (hunk 6 above)
Comment (lukasmas) [reply module-db/Interface/ContactRecord.cpp]: fixed

**Pull request merged**